src/libs/OptionsListUtils.js

```diff
@@ -92,11 +93,14 @@ function createOption(personalDetailList, report, draftComments, activeReportID,
             : '')
         + _.unescape(report.lastMessageText)
         : '';
+    const tooltipText = getReportParticipantsTitle(lodashGet(report, ['participants'], []));
 
     return {
         text: report ? report.reportName : personalDetail.displayName,
         alternateText: (showChatPreviewLine && lastMessageText) ? lastMessageText : personalDetail.login,
         icons: report ? report.icons : [personalDetail.avatar],
+        tooltipText,
+        participantsList: personalDetailList,
```

Comment:
Why is this being returned? Since it was already passed to this method as a param, wouldn't it already be available?

Reply:
We are creating options for different parts of the app and this method gives the object being used at data on those parts. This method is being called where the personalDetailList may be coming differently so as it is being passed down to this function and which as used to calculate other things 
https://github.com/Expensify/Expensify.cash/blob/951f9710f9cf527d4ce2f2715561884718941673/src/libs/OptionsListUtils.js#L82-L96,

We need emails and participants' names for the chat title so I have just used the details which contain the Name and email and avatar.